src/pages/api/add-trello-card.ts

```diff
@@ -107,14 +125,49 @@ class TrelloClient {
         return
     }
 
-    private async getLabelId(board: string, labelName: string): Promise<string | null> {
-        const labels = await this.getLabelsFromBoard(board)
+    private async getLabelById(labels: TrelloLabel[], labelName: string): Promise<string> {
         const label = labels.find((label) => label?.name?.toLowerCase() === labelName?.toLowerCase())
 
+        if (label == null) {
+            const idBoard = labels[0].idBoard
+            const createdLabel = await this.createLabel(idBoard, labelName, getRandomColor())
+
+            return createdLabel.id
+        }
+
         return label?.id ?? null
```

Comment:
```suggestion
        return label.id
```
id kan ikkje vere null her?